atomics/T1003/T1003.yaml

```diff
@@ -153,7 +153,7 @@ atomic_tests:
     The memory of lsass.exe is often dumped for offline credential theft attacks. This can be achieved with Sysinternals
     ProcDump.
 
-    Upon successful execution, you should see a file the following file created C:\Windows\Temp\lsass_dump.dmp.
+    Upon successful execution, you should see the following file created C:\Windows\Temp\lsass_dump.dmp.
```

Comment:
Test saves to `C:\Users\research\AppData\Local\Temp\lsass-comsvcs.dmp`  per env $TEMP.